kroxylicious-kms/src/main/java/io/kroxylicious/kms/service/UnsupportedRequestException.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
This file is in the wrong package, it is a concern of the test facade, not the production implementation.

Reply:
Moved to `kms-test-support` package